### PR TITLE
Feat/add signout feature

### PR DESCRIPTION
**Pull request by sahsisunny**
### Issue

Sign Out option in Navbar #234

### What is the change?

1. Add a sign out and My Profile option in the navbar with dropdown menu.
2. Pages - index.html, faq.html, discord.html

### Is Development Tested?

- [x] Yes
- [ ] No

### After Change Screenshots

https://user-images.githubusercontent.com/70854507/210047021-8eb3890d-0201-4d50-91fb-6a775d133bc2.mp4

### Final Output

https://user-images.githubusercontent.com/70854507/213382194-7aa2df2b-88c9-408e-b745-1dab66e14361.mp4

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -16,6 +17,19 @@
   </head>
 
   <body>
+    <div class="dropdown hide">
+      <ul class="dropdown-list">
+        <li class="dropdown-item">
+          <a class="dropdown-link" href="https://my.realdevsquad.com/">
+            My Profile
+          </a>
```
Comment (rohan09-raj) [inline faq.html:25]: Can we please check that the click is working everywhere on the option not only the text?

File: index.html [hunk 2]
```diff
@@ -32,6 +32,19 @@
   </head>
 
   <body>
+    <div class="dropdown hide">
+      <ul class="dropdown-list">
+        <li class="dropdown-item">
+          <a class="dropdown-link" href="https://my.realdevsquad.com/">
+            My Profile
+          </a>
```
Comment (rohan09-raj) [inline index.html:40]: Can we please check that the click is working everywhere on the option not only the text?

File: discord.html [hunk 3]
```diff
@@ -16,6 +17,19 @@
   </head>
 
   <body>
+    <div class="dropdown hide">
+      <ul class="dropdown-list">
+        <li class="dropdown-item">
+          <a class="dropdown-link" href="https://my.realdevsquad.com/">
+            My Profile
+          </a>
```
[the remark this answers is not in the document]
Comment (sahsisunny) [reply discord.html]: Sure, we can perform a thorough testing to ensure that the click function is working correctly on all parts of the option, not just the text. Once we have confirmed that everything is working as expected, we can say that the problem has been resolved.

File: index.html (hunk 2 above)
Comment (sahsisunny) [reply index.html]: Sure, we can perform a thorough testing to ensure that the click function is working correctly on all parts of the option, not just the text. Once we have confirmed that everything is working as expected, we can say that the problem has been resolved.

File: faq.html (hunk 1 above)
Comment (sahsisunny) [reply faq.html]: Sure, we can perform a thorough testing to ensure that the click function is working correctly on all parts of the option, not just the text. Once we have confirmed that everything is working as expected, we can say that the problem has been resolved.

File: js/navbar.js [hunk 4]
```diff
@@ -8,10 +11,43 @@ window.addEventListener('DOMContentLoaded', (event) => {
...
-const hamburger = document.querySelector('.hamburger');
-const navMenu = document.querySelector('.nav-menu');
 
 hamburger.addEventListener('click', () => {
   hamburger.classList.toggle('active');
   navMenu.classList.toggle('active');
 });
+
+const signout = () => {
+  fetch('https://api.realdevsquad.com/auth/signout', {
```
Comment (rohan09-raj) [inline js/navbar.js:21]: Can we please store this in a constant?

Comment (sahsisunny) [reply js/navbar.js]: done

File: js/navbar.js [hunk 5]
```diff
@@ -8,10 +11,43 @@ window.addEventListener('DOMContentLoaded', (event) => {
...
+  });
+};
+
+document.getElementById('signout-option').addEventListener('click', () => {
+  signout();
+});
+
+const myProfileButton = document.getElementById('my-profile');
+myProfileButton.addEventListener('click', () => {
+  window.location.href = 'https://my.realdevsquad.com/';
```
[the remark this answers is not in the document]
Comment (sahsisunny) [reply js/navbar.js]: done

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**